test/returns_dict.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Lawrence Livermore National Security, LLC and other CLIPPy
+// Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: MIT
+
+#include <clippy/clippy.hpp>
+#include <unordered_map>
+
+int main(int argc, char **argv) {
+  clippy::clippy clip("returns_dict", "Tests returning a dict");
+  clip.returns<size_t>("The Int");
```

Comment:
size_t needs to be unordered_map<std::string, int>

Reply:
Fixed - thanks. For some reason it worked though.